CONTRIBUTING.md

```diff
@@ -104,50 +104,32 @@ If you're **new to testing**, please review existing test files for examples
 of how to create tests.
 
 **Don't let the tests keep you from submitting your contribution!**
-If you're not sure how to do this or are having trouble, submit your pull
-request anyway. We will help you create the tests and sort out any kind of problem during code review.
+If you're unsure how to do this or are having trouble, submit your pull
+request anyway. We will help you create the tests and sort out any problems during the code review.
 
-You can learn more about our test suite in the development guide. However,
-if you wish to run the test suite locally, you can use:
-
-```bash
-make test
-```
+You can learn more about [how to run our test suite in the development guide, here](https://stravalib.readthedocs.io/en/latest/contributing/development-guide.html#about-the-stravalib-test-suite).
 
 ### Test coverage
 
-We use `codecov` implemented through the `pytest-cov` extension to sphinx to
+We use `codecov`, implemented through the `pytest-cov` extension to sphinx to
 track `stravalib`'s test % coverage. When you submit a pull request, you will
 see how that pull request affects our package's total test coverage.
 
 ### Documentation
 
 Our documentation is in the `doc` folder.
 We use [sphinx](https://www.sphinx-doc.org/) to build the web pages from
-these sources. To build the HTML files:
-
-```bash
-make -C docs html
-```
-
-To serve documentation locally use:
-
-```bash
-make -C docs serve
-```
-
-You can learn more about our documentation infrastructure in our
-[development guide](https://stravalib.readthedocs.io/en/latest/contributing/development-guide.html).
+these sources. You can learn more about [building and contributing to our
```

Comment:
I keep finding dated information in our readme. so let's just always update the docs and link to them from the readme rather than having things in two places :) I forget all of the places to update things!!